package.json

```diff
@@ -45,7 +45,7 @@
     "sinon": "^10.0.1",
     "split2": "^3.2.2",
     "standard": "^16.0.3",
-    "tap": "^14.10.8"
+    "tap": "^15.0.2"
```

Comment:
considering that 90% of the changes are tap-related, you could have considered keeping the tap upgrade out of this change and do it standalone instead